docs/references.rst

```diff
@@ -4,19 +4,56 @@ Ubuntu Pro Client reference
 ***************************
 
 Our reference section contains technical information for the Ubuntu Pro Client.
-This includes details of the network requirements, API definitions, and other
-related tools.
 
-Reference
-=========
+System and other requirements
+=============================
+
+The Ubuntu Pro Client is lightweight and has very few requirements. 
+
+* :ref:`Network requirements <network_requirements>`
+
+Interacting with the Client
+===========================
+
+There are two main ways to interact with the Pro Client: on the command line
+interface (CLI) in your terminal, or automatically via the API.
+
+* :ref:`CLI reference guide <commands>`
+* :ref:`API reference guide <api>`
+
+You may, in a production environment, also wish to test upcoming changes. For
+this, you can use the available Personal Package Archives (PPA) for the Ubuntu
+Pro Client.
```

Comment:
I see what you're saying, but technically you would want to test _not_ in a production environment. 
```suggestion
You may, in a staging or canary environment, also wish to test upcoming changes. For
this, you can use the available Personal Package Archives (PPA) for the Ubuntu
Pro Client.
```

Reply:
Yes, you're quite right - that was imprecise of me